CanHazFunny/CanHazFunny/Program.cs

```diff
@@ -1,10 +1,33 @@
-﻿namespace CanHazFunny;
+﻿using System;
+using System.Diagnostics.CodeAnalysis; // ADDED: Required for ExcludeFromCodeCoverage
+using System.Threading.Tasks; 
 
-class Program
+namespace CanHazFunny;
+
+// ADDED: Excludes this untestable file from the coverage report (Fixes 0% coverage)
+[ExcludeFromCodeCoverage] 
+sealed class Program
 {
     static void Main(string[] args)
     {
-        //Feel free to use your own setup here - this is just provided as an example
-        //new Jester(new SomeReallyCoolOutputClass(), new SomeJokeServiceClass()).TellJoke();
+        //Ask if they wanna hear a joke
+        Console.Clear();
+        Console.Write("Wanna hear a joke? (y/n): ");
+
+        //Get their answer
+        string answer = Console.ReadLine() ?? "n";
+        
+        //If they say no, exit program gracefully
+        if (!string.Equals(answer, "y", StringComparison.OrdinalIgnoreCase))
```

Comment:
Nitpick, could change the statement to 
```suggestion
        if (string.Equals(answer, "n", StringComparison.OrdinalIgnoreCase))
```
for potential code clarity. Entirely subjective of course.

Reply:
Thanks for the suggestion! I see your point. I'll take another look at it